### PR TITLE
Container Deployment: Language and grammar fixes.

### PR DESCRIPTION
**Pull request by iamAzeem**
Container Deployment: Language and grammar fixes.

Signed-off-by: Azeem Sajid <azeem.sajid@gmail.com>

### CODE REVIEW

**Comment by iamAzeem** [comment]
@repeatedly : Are you able to see the next commit on this [PR](https://github.com/fluent/fluentd-docs-gitbook/compare/1.0...iamAzeem:language-fixes-container-deployment-section) i.e. [93ca538](https://github.com/fluent/fluentd-docs-gitbook/commit/93ca538470bea64ce35c6f3d006f9ea32068b476)?

For me, it is able to merge but the last commit is not updated on this PR here.
The same is with the rest of these PRs.
I observed this when GitHub was unavailable earlier for about four (4) hours.

**Comment by repeatedly** [comment]
> Are you able to see the next commit on this PR i.e. 93ca538?

I can't see the commit on this PR... but I can see the diff via your link.

**Comment by repeatedly** [comment]
How about squash commits and force push?

**Comment by iamAzeem** [comment]
@repeatedly: Thanks! It worked.

**Comment by repeatedly** [comment]
Thanks!

**Comment by iamAzeem** [comment]
@repeatedly:

FYI:
I just observed that some links with line breaks in Markdown are not properly rendered.

Example:

docker-image-link-issue

I'll update all those in a single PR.

Regards!

**Comment by repeatedly** [comment]
I see. Line break is not supported in `[]`...

File: container-deployment/install-by-docker.md [hunk 1]
```diff
@@ -88,30 +94,41 @@ $ docker logs 775a8e192f2b | tail -n 1
...
-    -   [Store Apache Log into MongoDB](/guides/apache-to-mongodb.md)
-    -   [Data Collection into HDFS](/guides/http-to-hdfs.md)
 
 ------------------------------------------------------------------------
 
-If this article is incorrect or outdated, or omits critical information, please [let us know](https://github.com/fluent/fluentd-docs-gitbook/issues?state=open).
-[Fluentd](http://www.fluentd.org/) is a open source project under [Cloud Native Computing Foundation (CNCF)](https://cncf.io/). All components are available under the Apache 2 License.
+If this article is incorrect or outdated, or omits critical information, please
+[let us know](https://github.com/fluent/fluentd-docs-gitbook/issues?state=open).
+[Fluentd](http://www.fluentd.org/) is an open-source project under [Cloud Native
```
Comment (iamAzeem) [inline container-deployment/install-by-docker.md:132]: @repeatedly: Why is this working? It is being rendered okay.
Is it different from the rest of the page? Special case?

**Pull request merged**